charts/argo-cd/values.yaml

```diff
@@ -1535,6 +1535,17 @@ repoServer:
   #     - list
   #     - watch
 
+  # -- Init Container to copy argocd binary
```

Comment:
Drop the `--` from this line and rerun helm-docs.

Reply:
Done.